Header header
Obstacle_Global_Position[] obstacles_global_position
